lib/Controller/WopiController.php

```diff
@@ -580,9 +591,11 @@ public function putFile($fileId,
 	 * @return JSONResponse
 	 * @throws DoesNotExistException
 	 */
-	public function putRelativeFile($fileId,
-					$access_token) {
+	public function postFile(string $fileId, string $access_token): JSONResponse {
```

Comment:
What motivated this name change?

The postFile operation does not exist on Microsoft's WOPI spec
https://docs.microsoft.com/en-us/microsoft-365/cloud-storage-partner-program/rest/files/putrelativefile